setup.cfg

```diff
@@ -41,6 +41,8 @@ ignore = D100, D104, D401, W504, RST303, RST304, DAR103, DAR203
 per-file-ignores =
   # These function names are part of 3d party API:
   wemake_python_styleguide/visitors/ast/*.py: N802
+  # This module contains many visitors
+  wemake_python_styleguide/visitors/ast/compares.py: N802, WPS202
```

Comment:
If we have too many visitors, we should refactor our module 🙂 
Not just hide it!

Reply:
I guessed so, but I couldn't just do that on my own without asking about it first. I'd need to know which ones I should move and how I should call the new visitors module.